src/native_main.nim

```diff
@@ -73,6 +75,36 @@ proc findUserConfigFile(): Option[string] =
 
     return config_path
 
+proc debug_log(msg: string) =
+    if DEBUG:
+        write(stderr, msg)
+
+proc removeMacOSFileAttribute(file, attr: string): int =
+    var xattrErr = 0
+    var xattrCmd = ""
+
+    debug_log(">> xattrCmd == " & $xattrCmd & "\n")
+
+    xattrCmd = quoteShellCommand(["xattr", "-d", attr, file])
+
+    xattrErr = execCmd(xattrCmd)
+    return xattrErr
```

Comment:
If we don't need this function, can it be removed? Feel free to leave it in the commit history so we can find it again if we do need it.

```suggestion
```